### ISSUE TITLE
Remove docs/icons/Illustrator files

### ISSUE THREAD

**Issue by aendrew** [ft-interactive/visual-vocabulary-templates#119]
This is like ~100mb of AI files that is making cloning the repo brutally slow, almost to the point that people are going to think Vocab is broken on the initial clone.

Solutions:
- Move the large files to GitHub LFS (Large File Storage)
- Move the AI files to a separate repo

**Comment by aendrew** [comment]
I've discussed this with @sdbernard a bit, and although it's not overly painful to move everything to GitHub LFS, it *will* involve rewriting the commit history, which will break everyone's local repo. 

Thus, everyone will need to re-clone after we do this. I'm thus putting it off until @alansmithy returns so we can figure out how to do this in the least interruptive way possible.

**Comment by bobhaslett** [comment]
Have we decided what to do with this? I suggest we move them to them to their own folder in the Graphics-resources VVT folder

**Comment by aendrew** [comment]
@bobhaslett Even if we move them out of the Git repo, we'll still need to rewrite history to remove them fully. It's possible the solution will ultimately be just to have everyone delete the repo and reclone after we fix this.

**Comment by bobhaslett** [comment]
Has this issue been resolved, I thinkl we just ended up with the SVG files. Can we close this?

**Comment by aendrew** [comment]
Nope, we still need to rewrite history at some point to fully remove the files from the repo.
